src/python/openvino_genai/py_openvino_genai.pyi

```diff
@@ -1678,12 +1678,12 @@ class Tokenizer:
         Decode a batch of tokens into a list of string prompt.
         """
     @typing.overload
-    def encode(self, prompts: list[str], add_special_tokens: bool = True) -> TokenizedInputs:
+    def encode(self, prompts: list[str], add_special_tokens: bool = True, pad_to_max_length: bool = False, max_length: int | None = None) -> TokenizedInputs:
```

Comment:
I suppose these default arguments can overrides values stored in XML files?

E.g. `CLIPtokenizer` pads to 77 (its max_length), while doing the same via GenAI will use `pad_to_max_length: bool = False` ?

Reply:
This union notation for typing was added in Python 3.10: https://docs.python.org/3/whatsnew/3.10.html#pep-604-new-type-union-operator
We should use `Optional` until we drop support for 3.8 and 3.9

Reply:
> This default value is aligned with the default value that is written into IR.

Someone could change rt_info manually.

Reply:
@pavel-esir ok, thank you. Looks like I have mixed Clip encoder (which always expect 77 as input len) and tokenizer (which does not perform padding)

> Someone could change rt_info manually.

I suppose it's not in rt_info, but in `.bin` file